roles/openshift_health_checker/test/logging_check_test.py

```diff
@@ -98,21 +98,19 @@ def execute_module(module_name, *_):
     assert expect in str(excinfo)
 
 
-groups_with_first_master = dict(masters=['this-host', 'other-host'])
-groups_with_second_master = dict(masters=['other-host', 'this-host'])
-groups_not_a_master = dict(masters=['other-host'])
+groups_with_first_master = dict(oo_first_master=['this-host'])
+groups_not_a_master = dict(oo_first_master=['other-host'], oo_masters=['other-host'])
```

Comment:
Isn't `oo_first_master` necessarily in `oo_masters`?

Reply:
Presumably (during normal operation not a unit test). I'm not sure why I bothered putting `oo_masters` in there, nothing looks at it.